### PR TITLE
Add Notes on DB Creation To Reduce Confusion

### PR DESCRIPTION
**Pull request by romeara**
The container currently sets up databases in multiple ways - one uses behavior built-in to the base container, and isn't very discoverable to the uninitiated, and another that can be reasonably found for someone browsing the repository.

Add README notes specifying this behavior for future developers

### CODE REVIEW

File: README.md
Comment (paykin) [inline README.md]: I would add clarification that it is set inside Dockerfile .

**Review by paykin — APPROVED** [review]

File: README.md
Comment (romeara) [reply README.md]: Added. Note: I do not have permission to merge this, even once approved

**Pull request merged**
